common/src/main/cpp/extensions/openxr_fb_passthrough_extension_wrapper.cpp

```diff
@@ -202,43 +202,49 @@ void OpenXRFbPassthroughExtensionWrapper::_on_session_created(uint64_t p_session
 }
 
 void OpenXRFbPassthroughExtensionWrapper::_on_state_ready() {
-	XRInterface::EnvironmentBlendMode blend_mode = get_blend_mode();
-	if (blend_mode == XRInterface::XR_ENV_BLEND_MODE_ALPHA_BLEND) {
-		start_passthrough_layer(LAYER_PURPOSE_RECONSTRUCTION);
+	if (fb_passthrough_ext) {
```

Comment:
nit - to minimize visual noise due to the added indentation, we can instead do:

```
if (!fb_passthrough_ext) { 
    return;
}
```

Reply:
yeah that makes sense